src/components/v2/V2Create/forms/TokenForm/index.tsx

```diff
@@ -215,6 +231,11 @@ export default function TokenForm({
      * e.g. permyriads, parts-per-billion etc.
      * and NOT percentages.
      */
+    dispatch(
+      editingV2ProjectActions.setWeight(
+        weight ?? DEFAULT_ISSUANCE_RATE.toString(),
```

Comment:
**issue**: this needs to have the correct decimals (see [here](https://github.com/jbx-protocol/juice-interface/blob/3a0cfd99fa969f95c8cb6d2f95ea5602edecfbf6/src/redux/slices/editingV2Project.ts#L56) for how we do it currently).

Lets create 2 utility functions in `utils/v2/math`:
- `issuanceRateFrom`
- `formatIssuanceRate`

...that convert to and from `wei` (just like redemptionRateFrom etc.)

The issuance rate that ends up in redux must be the `issuance * 10e18` value

Let me know if that doesn't make sense

Reply:
Ok cool, will get onto this